blog/content/post/2020-07-27-coreboot_blogpost.md

```diff
@@ -0,0 +1,111 @@
+---
+title: coreboot Fundamentals Training
+abstract: 'coreboot Fundamentals Training is the first step of 3mdeb Firmware Training
+path. It provides solid grounds for further development towards proficiency in
+coreboot firmware platform enginering. With this blogpost we are starting coreboot fundamentals mini-course.'
+cover: /covers/coreboot_fundamentals_training.png
+author: kamila.banecka
+layout: post
+published: true
+date: 2020-07-27
+archives: "2020"
+
+tags:
+  - training
+  - coreboot
+  - BIOS
+categories:
+  - Firmware
+  - Miscellaneous
+
+---
+
+#### [coreboot Fundamentals Training](https://training.3mdeb.com/courses/coreboot-fundamentals)
+
+It's hard to find a reliable source from which developers could draw and quickly
+acquire knowledge substantial for fluent cooperation with coreboot. Relying on
+official documentation and extensive practice is obviously the best way to
+become proficient –  we went this way ourselves – unfortunately, it is
+time-consuming, especially when the new employee is joining the team and he
+needs to be quickly introduced with the coreboot-based project. Based on years
+of experience, vast knowledge, and a passion towards open source solutions,
+we've responded to this need by providing reliable and professional training
+that covers essential basics. We are proud of the way we've managed to hammer
+out years of experience into modular, easy-to-follow 4-hours training. **In this
+new blogpost series we will be publishing coreboot fundamentals mini-course**
+equipped with coreboot essentials. [coreboot Fundamentals
+Training](https://training.3mdeb.com/courses/coreboot-fundamentals) is the first
+step of 3mdeb Firmware Training path. It provides solid grounds for further
+development towards proficiency in coreboot firmware platform enginering.
+
+![graf](/blog/static/img/training_path.png)
```

Comment:
The paths should be just `img/...`

As it is now, the images are not rendering. Fix all the occurrences please

Reply:
done